src/app/ui/solution/solution.component.html

```diff
@@ -9,15 +9,19 @@
     <h6 class="mt-5">Available from pending scan</h6>
   </div>
   <div class="card-img-overlay align-items-center d-flex justify-content-center">
-    <h2 class="card-title text-center">
+    <h2 class="card-title text-center" *ngIf="solution.grade >= 0">
       {{solution.grade}} / {{question.total_grade}}
     </h2>
+    <h4 class="card-title text-center" *ngIf="solution.grade === -1">
```

Comment:
Why not use `*ngif="grade >= 0; else noGrade"` on the H2
and then add `#noGrade` on a ng-template surrounding the H4?

Reply:
That's why CR is great. I was lazy. Thanks.